google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
         List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
         postData.add(new BasicNameValuePair("tid", analyticsId));
 
-        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
```

Comment:
Don't worry about Android Studio.  I checked their code-base, they do not
use the UsageTracker  interface outside of our plugin.

On Thu, Jun 23, 2016 at 10:51 AM, Chanseok Oh <notifications@github.com>
wrote:

> In
> google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/754#discussion_r68249162>
> :
>
> > @@ -81,17 +81,15 @@ public void trackEvent(@NotNull String eventCategory,
> >          List<BasicNameValuePair> postData = Lists.newArrayList(analyticsBaseData);
> >          postData.add(new BasicNameValuePair("tid", analyticsId));
> >
> > -        postData.add(new BasicNameValuePair("cd19", eventCategory));  // Event type
>
> I mean we can do that with your new design, but not with this interface to
> be backward-compatible.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/754/files/a9c11a4f5a3db7e0fb75e48bff6e987e090f46ec#r68249162>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HWAzdgUl--GgTKn4kIsskpGyO1zkks5qOp1agaJpZM4I8SzV>
> .
>